src/rpc/blockchain.cpp

```diff
@@ -399,6 +399,7 @@ static std::vector<RPCResult> MempoolEntryDescription() { return {
     RPCResult{RPCResult::Type::ARR, "spentby", "unconfirmed transactions spending outputs from this transaction",
         {RPCResult{RPCResult::Type::STR_HEX, "transactionid", "child transaction id"}}},
     RPCResult{RPCResult::Type::BOOL, "bip125-replaceable", "Whether this transaction could be replaced due to BIP125 (replace-by-fee)"},
+    RPCResult{RPCResult::Type::BOOL, "unbroadcast", "Whether this transaction is currently unbroadcast (initial broadcast not yet confirmed)"},
```

Comment:
Suggested rephrasing: s/confirmed/acknowledged/.  I think "confirmed" might confuse the user into thinking we're referring to the transaction being included in a block.  Perhaps "acknowledged" would be less confusing, or if a few more words are ok, "(initial broadcast not yet acknowledged by any peers)".  Maybe @amitiuttarwar can squeeze this in #18807 (if not, I can open a separate PR).

Reply:
great suggestion. adding to #18807

Reply:
done